classification-example.sh

```diff
@@ -27,7 +27,12 @@ mkdir -p "${DATADIR}"
 
 if [ ! -f "${DATADIR}/dbpedia.train" ]
 then
-  wget -c "https://googledrive.com/host/0Bz8a_Dbh9QhbQ2Vic1kxMmZZQ1k" -O "${DATADIR}/dbpedia_csv.tar.gz"
+  if [ "`command -v wget`" != "" ]
+  	then
+  		wget -c "https://8c47f35de2544a202d525720f08188d254682381.googledrive.com/host/0Bz8a_Dbh9QhbQ2Vic1kxMmZZQ1k" -O "${DATADIR}/dbpedia_csv.tar.gz"
```

Comment:
`wget(1)` will follow the URL redirect automatically, so the current command:

```shell
wget -c "https://googledrive.com/host/0Bz8a_Dbh9QhbQ2Vic1kxMmZZQ1k" -O "${DATADIR}/dbpedia_csv.tar.gz"
```
is fine.